pandas/__init__.py

```diff
@@ -60,11 +60,15 @@
 # extension module deprecations
 from pandas.util.depr_module import _DeprecatedModule
 
-json = _DeprecatedModule(deprmod='pandas.json', deprmodto='pandas.io.json.libjson')
```

Comment:
you might be able to remove the ``deprmodto`` argument entirely (which I added to handle moving modules).